src/tests/getOneAccom.test.js

```diff
@@ -0,0 +1,44 @@
+import Chai from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../index';
+
+Chai.use(chaiHttp);
+Chai.should();
+
+describe('Barefoot nomad should allow multi city trips', () => {
+  it('It should login successfuly', (done) => {
+    Chai
+      .request(app)
+      .post('/api/v1/auth/signin')
+      .send({
+        email: 'blaisefr@gmail.com',
+        password: 'password',
+      })
+      .end((err, res) => {
+        res.body.should.have.status(200);
+        done();
+      });
+  });
+
+  it('It should get one accommodation with the given id', (done) => {
```

Comment:
```suggestion
  it('should get one accommodation with the given id', (done) => {
```
@AbdoulNuru There is no need of two ``it`` subjects

Reply:
@hakoemmy yeah sure.